coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -304,7 +319,7 @@ export default {
       const url = filing.paymentToken + '/receipts'
       const data = {
         corpName: this.entityName,
-        filingDateTime: filing.filingDate, // TODO: format as needed
+        filingDateTime: this.convertISOToCommonTime(filing.filingDateTime), // TODO: format as needed
```

Comment:
Is this now formatted as needed? Just wondering if the comment can be deleted or if more formatting is required (and should it be done here/now)?